utils/fs/fsutils.go

```diff
@@ -52,7 +53,9 @@ func FindFileInProc(inode uint64, suffix string) (string, error) {
 				continue
 			}
 			if tryInode == inode {
-				return filename, nil
+				if cmdline, err := GetCmdline(name); err == nil && strings.Contains(cmdline, "pause") {
```

Comment:
This changes the behaviour of the function. It is no longer a simple FindFileInProc. It checks the cmd line for a magic "pause" string. I don't think this is a reliable way to solve the issue
For example What will happen if we have a pod with 2 apps + 1 init container?
Do we have other options?

Reply:
According to kubernetes documentation pod always starts `pause` container with pid=1. Pause container has same lifetime as pod that it works on. So it doesn't matter how many apps and init containers we have in our pod. 
But yes, you are right, I have to choose another name for that function :)